charts/prl-citizen-frontend/values.aat.template.yaml

```diff
@@ -2,9 +2,9 @@ nodejs:
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
-  devmemoryLimits: 2Gi
-  devmemoryRequests: 512Mi
+  devmemoryRequests: 256Mi
```

Comment:
is it still required to be specified explicit in AAT?

Reply:
Yes I think @vivek-sekhar. 
Defaults for other envs are less than what we have defined here.
memoryRequests: 512Mi
cpuRequests: 25m
memoryLimits: 512Mi
cpuLimits: 500m